Dynatrace/dynatrace-operator
# Allow custom version for cloudnative

## Description

Currently the version field is ignored for CloudNativeFullStack deployments. This PR enables the custom version for CloudNativeFullStack.

[Jira](https://dt-rnd.atlassian.net/browse/DAQ-2271) 

## How can this be tested?

- Deploy CloudNativeFullStack with a custom OneAgent version set
  - OneAgent daemonset should have the custom version set + CodeModules should use the correct version (hence the CSI driver needs to download the correct version too) 
- Deploy CloudNativeFullStack with custom OneAgent version + custom CodeModules image set
  - the CodeModules image should be preferred
